src/streaming-client.ts

```diff
@@ -49,7 +50,11 @@ export class RevAiStreamingClient extends EventEmitter {
         this.baseUrl = `wss://api.rev.ai/speechtotext/${version}/stream`;
         this.requests = new PassThrough();
         this.responses = new PassThrough({ objectMode: true });
-        this.client = new client({ keepalive: true, keepaliveInterval: 30000 });
+        this.client = new client({
+            // @ts-ignore
+            keepalive: true,
+            keepaliveInterval: 30000
+        });
```

Comment:
Needed to add `@ts-ignore` because the typings for `ClientConfig` for the `websocket` package does not include `keepalive` or `keepaliveInterval` but they are still used - https://github.com/theturtle32/WebSocket-Node/blob/574b703ca978709832c6c2f67d6c1de376d432b5/lib/WebSocketConnection.js#L111